mmtk/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "mmtk_openjdk"
 version = "0.19.0"
 authors = [" <>"]
-rust-version = "1.66.0"
+rust-version = "1.71.1"
```

Comment:
This is the minimal Rust version, not the version we use for building and running tests. MSRV should be 1.70, and the version we use to run tests should be 1.71.1 (in the `rust-toolchain` file).

Reply:
Gotcha. Fixed it!